Core/GDCore/IDE/Events/EventsVariableInstructionTypeSwitcher.h

```diff
@@ -46,8 +55,13 @@ class GD_CORE_API EventsVariableInstructionTypeSwitcher
 
   const gd::Platform &platform;
   const gd::VariablesContainer &targetVariablesContainer;
-  gd::String objectName;
+  /**
+   * Groups don't have VariablesContainer
+   */
+  const gd::String groupName;
```

Comment:
This states a fact but does not explain the reason for the variable. Be more precise:
```suggestion
   * Groups don't have VariablesContainer, so [targetVariablesContainer will be pointing to nullVariablesContainer and we use a group name instead to check for....??]
   */
  const gd::String groupName;
```